onelogin_rules/onelogin_brute_force_by_username.yml

```diff
@@ -15,7 +15,13 @@ Description: A OneLogin user was denied access more times than the configured th
 Threshold: 10
 DedupPeriodMinutes: 10
 Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
-Runbook: Analyze the IP they came from, and other actions taken before/after.  Check if this user eventually authenticated successfully. 
+Runbook: Analyze the IP they came from, and other actions taken before/after.  Check if this user eventually authenticated successfully.
+SummaryAttributes:
+  - event_type_id
+  - actor_user_name
+  - user_name
```

Comment:
For this rule it appears we are just deduping based on the `user_name`, not sure if we need the `actor_user_name`.

Reply:
fixed